### PR TITLE
chore(release): 11.4.0

### PR DESCRIPTION
**Pull request by CurlyBracketEffect**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,22 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+## [11.4.0](https://github.com/paritytech/substrate-api-sidecar/compare/v11.3.18..v11.3.19) (2022-05-18)
```
Comment (TarikGul) [inline CHANGELOG.md:5]:
```suggestion
## [11.4.0](https://github.com/paritytech/substrate-api-sidecar/compare/v11.3.19..v11.4.0) (2022-05-18)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,22 @@
...
+
+**Upgrade Priority** Low
+
+### Bug Fixes
+
+- feat(balance-info): add query param to convert free balance to human ([#914](https://github.com/paritytech/substrate-api-sidecar/pull/914)) ([f1e03d6](https://github.com/paritytech/substrate-api-sidecar/commit/f1e03d6c771d74cd4848975c2ebdf42bd9ab8d8b))
```
Comment (TarikGul) [inline CHANGELOG.md:11]: This is a feature and should be under `### Features`

**Review by TarikGul — APPROVED** [review]
👍

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,25 @@
...
+
+**Upgrade Priority** Low
+
+### Features
+
+- feat(balance-info): add query param to convert free balance to human ([#914](https://github.com/paritytech/substrate-api-sidecar/pull/914)) ([f1e03d6](https://github.com/paritytech/substrate-api-sidecar/commit/f1e03d6c771d74cd4848975c2ebdf42bd9ab8d8b))
+
+### Bug Fixes
+
+- fix(deps): update pjs common, api, type interfaces ([#918](https://github.com/paritytech/substrate-api-sidecar/pull/918)) ([6baf526](https://github.com/paritytech/substrate-api-sidecar/commit/6baf526b0627b3d5ac662946a9403db72b53a617))
```
Comment (dvdplm) [inline CHANGELOG.md:15]:
```suggestion
- fix(deps): update polkadot.js common, api, type interfaces ([#918](https://github.com/paritytech/substrate-api-sidecar/pull/918)) ([6baf526](https://github.com/paritytech/substrate-api-sidecar/commit/6baf526b0627b3d5ac662946a9403db72b53a617))
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -2,6 +2,25 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+## [11.4.0](https://github.com/paritytech/substrate-api-sidecar/compare/v11.3.19..v11.4.0) (2022-05-18)
+
+**Upgrade Priority** Low
+
+### Features
+
+- feat(balance-info): add query param to convert free balance to human ([#914](https://github.com/paritytech/substrate-api-sidecar/pull/914)) ([f1e03d6](https://github.com/paritytech/substrate-api-sidecar/commit/f1e03d6c771d74cd4848975c2ebdf42bd9ab8d8b))
```
Comment (dvdplm) [inline CHANGELOG.md:11]: convert to human?

Comment (niklasad1) [reply CHANGELOG.md:11]: the PR description was too long `Returns the any atomic balance as a human readable number based on the given chains token decimal`

File: CHANGELOG.md (hunk 3 above)
Comment (CurlyBracketEffect) [reply CHANGELOG.md:15]: I don't mind changing it, but using recent commits as an example we have been shortening polkadot.js down to pjs.

File: CHANGELOG.md (hunk 4 above)
Comment (CurlyBracketEffect) [reply CHANGELOG.md:11]: I simply copied the commit name as it was found here https://github.com/paritytech/substrate-api-sidecar/pull/914 should I add "readable format" to the end so it reads, "add query param to convert free balance to human readable format"?

Comment (niklasad1) [reply CHANGELOG.md:11]: That makes sense but as the `CHANGELOG` is used for humans we should aim to a descriptive as possible.
We should do a better job with better descriptions in the future, sorry

File: CHANGELOG.md (hunk 3 above)
Comment (niklasad1) [reply CHANGELOG.md:15]: I agree with David here, internally we use `p.js` but probably not as clear to outside folks.

File: CHANGELOG.md (hunk 4 above)
Comment (CurlyBracketEffect) [reply CHANGELOG.md]: Ok updated the Change log to reflect this feedback. Thanks folks

**Review by niklasad1 — APPROVED** [review]

**Comment by TarikGul** [comment]
Checked the updated changes LGTM 👍

**Pull request merged**